lib/services/commands/commandService.js

```diff
@@ -95,28 +88,32 @@ function markAsExpired(command) {
             constants.COMMAND_EXPIRED_MESSAGE,
             constants.COMMAND_STATUS_ERROR,
             typeInformation,
-            callback);
+            callback
+        );
     }
 
-    async.waterfall([
-        apply(deviceService.getDevice, command.deviceId, command.service, command.subservice),
-        getGroup,
-        calculateTypeInformation,
-        updateExpiredCommand
-    ], function(error) {
-        if (error) {
-            logger.error(context, 'Error updating polling command to expire: %j', error);
-        } else {
-            logger.debug(context, 'Command successfully expired');
+    async.waterfall(
+        [
+            apply(deviceService.getDevice, command.deviceId, command.service, command.subservice),
+            getGroup,
+            calculateTypeInformation,
+            updateExpiredCommand
+        ],
+        function (error) {
+            if (error) {
+                logger.error(context, 'Error updating polling command to expire: %j', error);
+            } else {
+                logger.debug(context, 'Command successfully expired');
+            }
         }
-    });
+    );
 }
 
 function expirationDaemon() {
     if (config.getConfig().pollingExpiration) {
-        config.getCommandRegistry().removeFromDate(
-            Date.now() - config.getConfig().pollingExpiration,
-            function(error, results) {
+        config
```

Comment:
I'm not sure if this way is more easy to read....

Reply:
Automated change - take it up with the guys at *Prettier*. :smile: This syntax has already been applied across the IoT Agents themselves the only hold out is the IoT-Node-lib so this is the one that isn't standardized.